server/pgserver.xml.in

```diff
@@ -339,7 +339,7 @@
         </runProgram>
         <runProgram>
             <program>launchctl</program>
-            <programArguments>unload /Library/LaunchDaemons/com.edb.launchd.${servicename}.plist</programArguments>
+            <programArguments>unload /Library/LaunchDaemons/${servicename}.plist</programArguments>
```

Comment:
any specific reason to change this? Usually all the .plist file begins with **com.**

Reply:
when i use this name with "com", it is giving error- No such fie or directory. When i gave this it ran fine

Reply:
but we have been using the previous name for a long time and it always worked